docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -303,7 +311,15 @@ On OpenProject inside the storage administration (*Administration → File stora
 
 ##### While using a self signed TLS/SSL certificate you receive "certificate verify failed"
 
-Some administrators setup Nextcloud using a self signed TLS/SSL certificate with their own CA (certificate authority). The CA needs to be known on the OpenProject server. On a Debian/Ubuntu based server, make sure you add the CA certificate for your Nextcloud certificate to `/usr/local/share/ca-certificates` and run `sudo update-ca-certificates` afterwards. Then OpenProject's Ruby code should be able to verify your Nextcloud TLS/SSL certificate when emitting HTTPS requests to your Nextcloud server.
+Some administrators setup Nextcloud using a self signed TLS/SSL certificate with their own CA (certificate authority). The CA needs to be known on the OpenProject server. On a Debian/Ubuntu based server, make sure you add the CA certificate for your Nextcloud certificate to `/usr/local/share/ca-certificates` and run `sudo update-ca-certificates` afterwards.
+
+Add the following line to the file `.env` in your application's home directory:
+
+```ENV
```

Comment:
@Kharonus

- I am not sure if `ENV` as a syntax actually exists.
- I am not sure if using an `.env` file is the right, the go-to approach to set ENV variables.

Reply:
I'd try not to be too specific and instead say:

````markdown
Add the following environmental variable to the OpenProject server and restart it:

```
SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
```
````

point is, that we do not want to contain every possible way out there to handle env variables. `.env` in this case is a file only interpreted by the docker daemon when composing configurations.